R/fit_model.R

```diff
@@ -0,0 +1,30 @@
+fit_model <- function(data, formula, user_supplied_args) {
+  UseMethod("fit_model")
+}
+
+#' @export
+fit_model.RtGam_gam <- function(data, formula, user_supplied_args) {
```

Comment:
Rather than hard coding arg.match earlier on you could have a default saying no known backend found (and how to find the method with backends). The reason to do this vs what you have is what you have precludes people adding their own backends without internal package changes. 

I think there is a low chance they actually would though so this is mostly taste and desire for control.

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/20/commits/17f48d47ba089e7728ede4031d80ec6de24385e9